Penteract/Assets/Scripts/VideoScene1Start.h

```diff
@@ -15,11 +16,15 @@ class VideoScene1Start : public Script
 	void Update() override;
 
 public:
+	static bool cameFromStartScene;
```

Comment:
Don't use statics, use GameplaySystems::GetGlobalVariable / GameplaySystems::SetGlobalVariable for those purposes. Is like the unity GameManager

Reply:
Substituted static bool with a GetGlobalVariable